js/main.js

```diff
@@ -76,3 +76,97 @@ document.addEventListener('DOMContentLoaded', () => {
+
+// *******Create data object*******
+
+const projectSection = [
+  {
+    img: './images/imageplaceholder.png',
+    title: 'Multi-Post Stories',
+    supportingText: 'A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    langBox: {
+      css: 'css', html: 'html', Bootstrap: 'Bootstrap', Ruby: 'Ruby',
+    },
+    button: 'See Project',
+  },
+  {
+    img: './images/imageplaceholder.png',
+    title: 'Multi-Post Stories',
+    supportingText: 'A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    langBox: {
+      css: 'css', html: 'html', Bootstrap: 'Bootstrap', Ruby: 'Ruby',
+    },
+    button: 'See Project',
+  },
+  {
+    img: './images/imageplaceholder.png',
+    title: 'Multi-Post Stories',
+    supportingText: 'A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    langBox: {
+      css: 'css', html: 'html', Bootstrap: 'Bootstrap', Ruby: 'Ruby',
+    },
+    button: 'See Project',
+  },
+  {
+    img: './images/imageplaceholder.png',
+    title: 'Multi-Post Stories',
+    supportingText: 'A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    langBox: {
+      css: 'css', html: 'html', Bootstrap: 'Bootstrap', Ruby: 'Ruby',
+    },
+    button: 'See Project',
+  },
+];
+
+const worksContainer = document.querySelector('#porto');
+projectSection.forEach((work, i) => {
+  const workCards = document.createElement('div');
+  workCards.classList.add('workCards');
+  workCards.innerHTML = `  
+    <div class="grid-container">
+        <div class="work-1 work">
+
+            <div class="col-1">
+                <img src=${work.img} alt="Image placeholder">
+            </div>
+
+            <div class="col-2">
+                <h3 class="title">${work.title}</h3>
+                <p class="supporting-text">${work.supportingText}</p>
+                <ul class="lang-box">
+                    <li>${work.langBox.css}</li>
+                    <li>${work.langBox.html}</li>
+                    <li>${work.langBox.Bootstrap}</li>
+                    <li class="border-check">${work.langBox.Ruby}</li>
+                </ul>
+                <div class="work-button"><button class="card-btn" type="button" data-work="${i}">${work.button}</button></div> 
+
+            </div>
+        </div>
+    </div>
+  `;
+
+  worksContainer.appendChild(workCards);
+});
+
+// ******* Modal Popup*******
+
+const modal = document.getElementsByClassName('modal');
```

Comment:
- [ ] [OPTIONAL] This is a peer-programming project, kindly in the future make sure to pay attention to this requirement, all the students in the group should contribute to this PR